lib/parse-form-data.ts

```diff
@@ -20,11 +20,16 @@ export function parseFormData(formValues: FormData) {
 			? (formValues.get("dateOfIssue") as string)
 			: Date.now(),
 	);
+	const trueCeremonyTime = new Date(
+		formValues.get("ceremonyTime")
+			? (formValues.get("ceremonyTime") as string)
+			: Date.now(),
+	);
 	const placeOfOrigin = formValues.get("placeOfOrigin")
 		? (formValues.get("placeOfOrigin") as string)
 		: "THE WOODS";
 
-	const sendToDb = formValues.get("sendToDb") as string | undefined;
+	const sendToDb = formValues.get("sendToDb") as unknown as boolean;
```

Comment:
huh why's this `as unknown`? that's usually a sign that something's going wrong with types

Reply:
Forgot this was there, I needed to clear this error
![image](https://github.com/user-attachments/assets/44bf6d91-4673-4bba-8070-e6d68521d3b1)

No, I cannot find any reason that's happening. It's always a boolean and always acts as such. Agh.

Reply:
whatever, we can figure it out later